pep/app/pods/browse/journal/index/route.ts

```diff
@@ -0,0 +1,44 @@
+import Route from '@ember/routing/route';
+
+import usePagination, { RecordArrayWithMeta } from '@gavant/ember-pagination/hooks/pagination';
+import { buildQueryParams } from '@gavant/ember-pagination/utils/query-params';
+
+import BrowseJournalIndexController from 'pep/pods/browse/journal/index/controller';
+import Journal from 'pep/pods/journal/model';
+import Source from 'pep/pods/source/model';
+
+export default class BrowseJournalIndex extends Route {
+    sourceCode?: string;
+    journalInformation?: Journal;
+    model() {
+        const journalParams = this.paramsFor('browse.journal') as { pep_code: string };
```

Comment:
continuing from the other comment, you'd then be able to reuse that params interface and import it here